code/game/objects/items/weapons/melee/misc.dm

```diff
@@ -228,6 +233,173 @@
 	new /mob/living/simple_mob/mechanical/cyber_horror(T)
 	return
 
+//Interestingly, a magic flame sword has a lot in common with the thermal cutter Tech and I made for Goblins. So I decided it should share some of that code.
+/obj/item/melee/ashlander
+	name = "bone sword"
+	desc = "A sharp sword crafted from knapped bone. In spite of its primitive appearance, it is still incredibly deadly."
+	icon_state = "bonesword"
+	hitsound = 'sound/weapons/bladeslice.ogg'
+	force = 20
+	throw_force = 10
+	w_class = ITEMSIZE_NORMAL
+	slot_flags = SLOT_BELT
+	attack_verb = list("attacked", "slashed", "stabbed", "sliced", "torn", "ripped", "diced", "cut", "chopped")
+	sharp = 1
+	edge = 1
+
+/obj/item/melee/ashlander/elder
+	name = "elder bone sword"
+	desc = "These swords are crafted from one solid piece of a gigantic bone. Carried by the Ashlander priesthood, these weapons are considered holy relics and are often preserved over the lives of their wielders."
+	icon_state = "elderbonesword"
+	var/active = 0
+	var/flame_intensity = 2
+	var/flame_color = "#FF9933"
+	var/SA_bonus_damage = 25 // 50 total against demons and aberrations.
+	var/SA_vulnerability = MOB_CLASS_DEMONIC | MOB_CLASS_ABERRATION
+
+/obj/item/melee/ashlander/elder/Initialize(mapload)
+	. = ..()
+	AddComponent(/datum/component/anti_magic, TRUE, TRUE, FALSE, null, null, FALSE)
+
+/obj/item/melee/ashlander/elder/afterattack(atom/A, mob/user as mob)
```

Comment:
remember to not copypaste "as x" from things
as thing is used in verb arguments
if a proc isn't a verb you don't need it at all

Reply:
Good catch. Removed.